controllers/news.controllers.js

```diff
@@ -82,3 +84,14 @@ exports.postCommentByArticleId = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.deleteCommentById = async (req, res, next) => {
+  try {
+    let { comment_id } = req.params;
+    await checkCommentIdExist(comment_id);
+    await removeCommentById(comment_id);
+    res.status(204).send({});

```

Comment:
if you don't want to send a body you can use the method

`res.sendStatus(204)`! 🚀